templates/index.html

```diff
@@ -27,6 +27,7 @@
   {% include "takeovers/_14-04-esm.html" %}
   {% include "takeovers/_ubuntu-core-full-disk-encryption.html" %}
   {% include "takeovers/_fin-serv-whitepaper-takeover.html" %}
+  {% include "takeovers/_german_compliance-webinar.html" %}
```

Comment:
```suggestion
  {% include "takeovers/_german-compliance-webinar.html" %}
```

wrong include name

![image](https://user-images.githubusercontent.com/441217/54212163-6b950b80-44da-11e9-823f-1592c684af72.png)